close_test.go

```diff
@@ -174,7 +176,8 @@ func TestCloseStress(t *testing.T) {
 	for i := 0; i < numHandlers; i++ {
 		wg.Add(1)
 		go func() {
-			testutils.WithTestServer(t, nil, func(ts *testutils.TestServer) {
+			opts := testutils.NewOpts()
```

Comment:
I don't - I just ran the same macro on the whole file without thinking about it. I'll back out the changes we don't need yet.